Makefile

```diff
@@ -9,33 +9,36 @@ _MOCHA=./node_modules/.bin/_mocha
 UGLIFY=./node_modules/.bin/uglifyjs
 NYC=./node_modules/.bin/nyc
 
-.PHONY: test test-all-nodejs all clean coverage
+.PHONY: test test-all-nodejs coverage lib/defs.js
 
-all: lib/defs.js
```

Comment:
I replaced the default "all" target with an "error" target because we don't want people running make and unintentionally generating new lib/defs.js